tests/conftest.py

```diff
@@ -164,6 +165,12 @@ def sleepless(monkeypatch):
     monkeypatch.setattr(time, "sleep", lambda s: None)
 
 
+@pytest_asyncio.fixture()
+async def clear_dl_queue():
+    await get_dl_queue().backend.clear()
+    yield
```

Comment:
Instead of this, I feel like it might be cleaner to do a dependency override for `get_dl_queue` where we use something like we have in the queue tests. Haven't tested this fully, but something like:

```python
@pytest.fixture
def queue(tmp_path):
    return DeadLetterQueue("file://" + str(tmp_path))

@pytest.fixture
def app(queue):
    app.dependency_overrides[get_dl_queue] = lambda: queue
    return jbi.app.app
```